package/autoyast2.changes

```diff
@@ -37,6 +44,23 @@ Thu Jul  6 14:31:28 CEST 2017 - schubi@suse.de
 - 3.1.159
 
 -------------------------------------------------------------------
+Fri Jun  9 09:08:50 CEST 2017 - schubi@suse.de
```

Comment:
I am not sure if this is the proper way to merge logs. I would expect all these changes to be at the top. Otherwise, I would think this change was included in version `3.1.159`, and it was not.